docs/installing-solidity.rst

```diff
@@ -300,6 +300,27 @@ CMake options
 
 If you are interested what CMake options are available run ``cmake .. -LH``.
 
+SMT Solvers
+-----------
+In some cases, installation may fail due to issues related to the SMT solvers
+integration. The dependency in the system may be met, but the linkage shows 
+incompatibilities in the interfaces used by Solidity and the ones provided
+by the system, erroring with ``undefined references`` and more.
+
+
+Inside the build folder you can disable them, since they are enabled by default:
+
+.. code:: bash
+
+    # disables only Z3 SMT Solver.
+    cmake .. -DUSE_Z3=false 
```

Comment:
these should be `ON` / `OFF` as per the cmake standard we're following

Reply:
Switched to ON / OFF.

Reply:
Actually ON/OFF is part of cmake so one can just use `if (USE_Z3)` 😉

Reply:
Right, done as well :)